### DISCUSSION TITLE
Welcome to winutil Discussions!

### DISCUSSION THREAD

**Discussion by ChrisTitusTech** [ChrisTitusTech/winutil#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tmcarter** [comment]
Thank you for putting this together!

**Comment by mperez738** [comment]
Where can i get the list of all the services that are set to manual?

**Comment by anonyao** [comment]
Hello my favorite debloater Guy !!.  Your debloat winutil, still failure. Now egdge dont remove. Cortana failed / msAPPS "people" dont gone, everytime its kinda new thing dont work:/ . Ive tried reset tweaks. Unibstalled and deleted all files and temps. Im forced to use unstabled Privacy.sexy 🤣

**Comment by chizy21** [comment]
Feature Request:  How can I update the paid version to the latest?  Or could you add an option to do this without having to repay to g et the newer version.

Thanks
Greg

**Reply by Perec1975** [reply]
I also purchased the paid version, the purchase was at the beginning of the 8th month. 2 weeks later I received an email. That a more recent version was released, this was maybe the 30th day of 8 months. In it there was a download link. The file downloaded from it was completely "with different looking content, a different, smaller size exe file" than the one I bought. Since then, the program has been updated twice, but I haven't received an email notification. It wasn't even in junk mail. Maybe we should make this email thing regular, or there should be some sort of freezing option. the code "CTT" also does not work in the purchase tab since I bought it in 2024. Is it currently the case that you pay once and then don't get an upgrade unless you pay again? I would be sad. This "paid" version is a bit confusing to me.

Thanks,
 Attila

**Reply by Zizuar** [reply]
That, or a built-in updater. Shouldn't be hard to have the exe file update itself since most of this seems headless.
